permission/db.go

```diff
@@ -8,9 +8,23 @@ import (
 // getPermissions returns all permissions
 func (s *Store) getPermissions(ctx context.Context) (p []Permission, err error) {
 	err = s.db.SelectContext(ctx, &p, `SELECT *
-		FROM people.permissions;`)
+		FROM people.permissions
+		ORDER BY name;`)
 	if err != nil {
 		return nil, fmt.Errorf("failed to get permissions: %w", err)
 	}
 	return p, nil
 }
+
+// getPermissions returns all permissions
```

Comment:
We have both the constants in `permission/permissions` and this - which is the source of truth for what are the valid permissions? Might be a bit out of scope of this PR, but I can see this causing confusion.

Reply:
The internal permissions are for things that are needed but the database gets the up to date ids and descriptions

Reply:
The database select is used for setting what permissions users have but the internal are reference permissions